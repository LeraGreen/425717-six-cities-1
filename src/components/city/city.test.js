import React from "react";
import renderer from "react-test-renderer";

import City from "./city.jsx";


it(`App is rendered correctly with active city`, () => {
  const tree = renderer
    .create(<City 
      city={`Paris`}
      isActive={true}
      onCityChange={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`App is rendered correctly with inactive city`, () => {
  const tree = renderer
    .create(<City 
      city={`Cologne`}
      isActive={false}
      onCityChange={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
