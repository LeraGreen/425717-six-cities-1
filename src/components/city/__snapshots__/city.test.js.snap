// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App is rendered correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Chelyaba
    </span>
  </a>
</li>
`;

exports[`App renders correctly 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    href="#"
    onClick={[Function]}
  >
    <span>
      Chelyaba
    </span>
  </a>
</li>
`;
